core/file/atime_spec.rb

```diff
@@ -15,6 +15,17 @@
     File.atime(@file).should be_kind_of(Time)
   end
 
+  platform_is :linux do
+    it "returns the last access time for the named file with microseconds" do
+      3.times do
```

Comment:
There is already only 1/1000 chance per iteration to get 0 (assuming millisecond granularity), so I think it's fine.
The second iteration will be even more unlikely to get 0 as the work in between will not take exactly one second.